eng/testing/performance/performance-setup.sh

```diff
@@ -285,6 +285,8 @@ if [[ "$internal" == true ]]; then
     else
         if [[ "$logical_machine" == "perfowl" ]]; then
             queue=Ubuntu.2204.Amd64.Owl.Perf
+        elif [[ "$logical_machine" == "perftiger_crossgen" ]]; then
+            queue=Ubuntu.1804.Amd64.Tiger.Perf
```

Comment:
This will need to change since it is taking us in the wrong direction.

Reply:
Do you mean remove this condition and just leave the `else` for `22.04 Tiger`?

Reply:
I just mean that the new condition enables an 1804 queue, which we don't want.

Reply:
This is needed, as context: https://github.com/dotnet/runtime/pull/90091

Some of the deps that are needed in perf testing don't work in 22.04

Reply:
@cincuranet, can you please review the perf changes to ensure the perf pipeline won't regress?

Reply:
@ilyas1974 -- When do you plan to retire the 1804 machines?

Reply:
Removed it now.

Reply:
@hoyosjs LGTM